src/multilevel_bisect/coarsen.cpp

```diff
@@ -1,436 +1,445 @@
-#include <algorithm>

```

Comment:
Something went wrong with git here I guess, as it thinks all lines were removed (and similar ones added). Do you have an idea how to fix this?